meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -44,8 +45,12 @@ do_install:append() {
             cat ${WORKDIR}/domu-vdevices-virtio-nogsx.cfg >> ${CFG_FILE}
         fi
     else
-        cat ${WORKDIR}/domu-vdevices.cfg >> ${CFG_FILE}
-        cat ${WORKDIR}/pvr-${XT_DOMU_CONFIG_NAME} >> ${CFG_FILE}
+        if ${@bb.utils.contains('MACHINE_FEATURES', 'gsx', 'true', 'false', d)}; then
+            cat ${WORKDIR}/domu-vdevices.cfg >> ${CFG_FILE}
```

Comment:
I'd better removed `gsx` from `domu-vdevices` and added `domu-vdevices-gsx. Idea is that you always include `domu-vdevices` and add `domu-devices-gsx` conditionally.